ml-proto/spec/float.ml

```diff
@@ -0,0 +1,145 @@
+module type RepresentationType =
+sig
+  type t
+
+  val nondeterministic_nan : t
+  val bits_of_float : float -> t
+  val float_of_bits : t -> float
+
+  val logand : t -> t -> t
+  val logor : t -> t -> t
+  val logxor : t -> t -> t
+
+  val min_int : t
+  val max_int : t
+
+  val size : int
+end
+
+module type S =
+sig
+  type t
+  type bits
+  val of_float : float -> t
+  val to_float : t -> float
+  val of_string : string -> t
+  val to_string : t -> string
+  val of_bits : bits -> t
+  val to_bits : t -> bits
+  val add : t -> t -> t
+  val sub : t -> t -> t
+  val mul : t -> t -> t
+  val div : t -> t -> t
+  val sqrt : t -> t
+  val min : t -> t -> t
+  val max : t -> t -> t
+  val ceil : t -> t
+  val floor : t -> t
+  val trunc : t -> t
+  val nearest : t -> t
+  val abs : t -> t
+  val neg : t -> t
+  val copysign : t -> t -> t
+  val eq : t -> t -> bool
+  val ne : t -> t -> bool
+  val lt : t -> t -> bool
+  val le : t -> t -> bool
+  val gt : t -> t -> bool
+  val ge : t -> t -> bool
+  val size : int
```

Comment:
Doesn't seem to be anymore. size was there a few commits ago when int cast was integrated in float, but I guess I just let it reappear. In any case, I removed it now. Thanks!